test/playlist-loader.test.js

```diff
@@ -791,6 +794,43 @@ QUnit.test('uses last segment duration for refresh delay', function(assert) {
     'used half targetDuration when update is false');
 });
 
+QUnit.test('isLowestEnabledRendition detects if we are on lowest rendition',
```

Comment:
I think putting these tests in `playlist.test.js` would be more appropriate and then you can test on just plain objects with the info thats needed instead of all this extra creating a loader, responding to request, etc.

Reply:
master-playlist-controller.test.js also already has integration tests for isLowestEnabledRendition, so even more reason to convert these to unit tests